docs/blog/newsletter-september-2017.rst

```diff
@@ -0,0 +1,83 @@
+#####################################
+Write the Docs Newsletter – September
+#####################################
+
+Hi everybody!
+
+Kelly O'Brien here, your friendly neighborhood newsletter editor, wishing you all a very happy September! As it happens, September is Be Kind to Editors & Writers Month (h/t @monique.semp for the discovery)! So make sure you take a moment to be kind to yourself and to the documentarians around you!
+
+Another exciting fact about September: It's the month that Write the Docs Prague happens! In fact, the conference is very nearly upon us! Folks have started to make their way to Prague (I'm hopping on a train this afternoon!), and the conference chatter is starting to pick up in the #wtd-conferences channel on Slack, if you're of a mind to tune in.
+
+In the meantime, though, we've had another month of interesting conversations and insights shared on the slack channel. Read on, and enjoy!
+
+*********************************
+Storing and Testing Code Snippets
+*********************************
+How do you work with code samples? Is it better to keep them in your doc source files or store them externally? How do you make sure your code examples are always up to date and working? In other words, how do you test them?
+
+Documentarians tackled this thorny question recently, and as usual the answer was "it depends." Nobody had a perfect solution, but we did come up with an excellent list of items to consider:
+
+* If you keep your snippets in your documentation, you need to export for testing. If the writing team works closely with tests, great, but more often you'll need to work with a separate dev team. Coordination and communication can get complicated (and therefore error-prone), but this approach is more reliable for figuring out what needs to change in the explanatory docs.
+
+* If you maintain your code examples externally, testing can be smoother, but keeping the docs up to date with changed examples is more challenging. This approach might work better for large sets of examples.
```

Comment:
again, "up-to-date" instead?

Reply:
good catch – seems to be one of those copyediting rules I can't keep in my head...